es-vue-base/src/lib-components/EsReviewModal.vue

```diff
@@ -26,6 +26,7 @@
                         :reviewer-id="review.reviewer_id"
                         :user-id="userId"
                         class="mb-4"
+                        :report-flag-visible="review.reportFlagVisible"
```

Comment:
```suggestion
                        :report-flag-visible="review.report_flag_visible"
```
This will transfer the value of the prop to this component!